cypress/e2e/Swagger-API/PetstoreAPI.cy.js

```diff
@@ -0,0 +1,30 @@
+/// <reference types='cypress' />
```

Comment:
Are you sure that this test file is a part of e2e?

Reply:
All tests must be stored in folder 2, otherwise they will not run in Cypress